### DISCUSSION TITLE
Unable to edit or unhide comment by closed acc. on issue

### DISCUSSION THREAD

**Discussion by YoshiRulz** [community/community#28814] [New to GitHub]
It seems my [previous thread on Discourse](https://github.community/t/unable-to-edit-or-unhide-comment-by-closed-acc-on-issue/162575) (dead link) wasn't migrated.
tl;dr: I have hide/edit perms for [this repo](https://github.com/TASEmulators/BizHawk). I cannot unhide or edit [this comment](https://github.com/TASEmulators/BizHawk/issues/1708#issuecomment-548456357) (3rd including OP) made by @ghost on [this Issue](https://github.com/TASEmulators/BizHawk/issues/1708).

**Comment by queenofcorgis** [comment]
👋🏼 @YoshiRulz - hoping I can help - are you trying to hide a specific comment - looks like you linked me to a Discussion here - if so, which one?

**Reply by YoshiRulz** [reply]
Ah, I didn't catch that I pasted the wrong link. Turns out the Copy Link button doesn't work on hidden comments. I've manually fixed the link.

**Reply by queenofcorgis** [reply]
Would you mind sharing a screenshot of what options you see when you try to unhide the comment please - I took a look at the comments and some are able to be hidden and unhidden, but that third one isn't letting me either.

**Reply by YoshiRulz** [reply]
[new screencast](https://youtu.be/B7qcijP1TA0)
I checked an earlier recording and I wasn't able to unhide the comment, but it seems I am now. Still can't edit any of them though.

**Reply by queenofcorgis** [reply]
Thanks for sharing - I'm going to ask some of my counterparts about how you can edit and/or why that is happening. Hang tight!
